materialize-firebolt/spec.go

```diff
@@ -23,7 +24,8 @@ func LoadSpec(cfg config, materialization string) (map[string]*pf.Materializatio
 	sess := session.Must(session.NewSession(&awsConfig))
 	downloader := s3manager.NewDownloader(sess)
 	buf := aws.NewWriteAtBuffer([]byte{})
-	existingSpecKey := fmt.Sprintf("%s%s.flow.materialization_spec", cfg.S3Prefix, materialization)
+
+	existingSpecKey := strings.TrimLeft(fmt.Sprintf("%s/%s.flow.materialization_spec", cfg.S3Prefix, materialization), "/")
```

Comment:
Ok, I guess this is what allows the default value to work?

I'm wondering if it might be better to just disallow both leading and trailing slashes? AFAIK it'd be really weird, but technically permissible, for an S3 object key to start with a `/`, so it might be confusing to show it in the UI and then strip off here. Up to you, but just wanted to raise it as something to consider.

Reply:
Tested this with `/test/blah/` as input, and  the files are put in `bucket/test/blah/...` as expected, so I think we are good 👍🏽